noxfiles/utils_nox.py

```diff
@@ -60,3 +61,34 @@ def teardown(session: nox.Session) -> None:
 def install_requirements(session: nox.Session) -> None:
     session.install("-r", "requirements.txt")
     session.install("-r", "dev-requirements.txt")
+
+
+@nox.session()
+def init_saas_connector(session: nox.Session) -> None:
```

Comment:
The nox command to create the connector files, it takes a connector ID as a param and creates all the required files